src/theme/theme.ts

```diff
@@ -611,7 +612,7 @@ export const lightTheme = createTheme(
       },
       text: {
         primary: '#000',
-        secondary: alpha(themeCustomized.palette.black.main, 0.75),
+        secondary: alpha(themeCustomized.palette.black.main, 0.48),
```

Comment:
@oktapodia Is there a specific need for this change? It messes with the color on articles as well - on both light + dark